src/bundle/Resources/public/scss/_card.scss

```diff
@@ -34,14 +34,16 @@
             padding: .25rem 1.25rem;
             justify-content: space-between;
             z-index: 1040;
+        }
+
+        .form-inline {
+            .btn-danger {
+                display: flex;
+                height: calculateRem(48px);
```

Comment:
Aren't properties like `align-items: center;` and `justify-content: center` missing here, are they?

Reply:
It works without these properties.